javascript/react/basics-i/the-component-lifecycle.md

```diff
@@ -39,21 +39,21 @@ These methods correspond to exact points in the **component's lifecycle**.
 
 A **lifecycle hook** is the `constructor(props)` which is commonly used to set the initial `state` of your component, but not only.
 
-Suppose you want to `log` a message when a component **will** be mounted[1] (before being `render`ed). For this, you can use the `componentWillMount` method:
+Suppose you want to `log` a message after a component was mounted[1] (after being `render`ed). For this, you can use the `componentDidMount()` method:
 ```jsx
 class Spawn extends React.Component {
-  componentWillMount() {
-    console.log('I will spawn');
+  componentDidMount() {
+    console.log("I have spawned");
   }
   render() {
     return <p>I'm basic</p>;
   }
 }
 ```
 
-This `component` can be dynamically **mounted** multiple times in an application, when this happens we'll see the `log` in console.
+This `component` can be **mounted** dynamically multiple times in an application, and after this happens we'll see the `log` in the console.
```

Comment:
no `component` code

Reply:
also no need for `log` I think, "log" would be enough

Reply:
applies to the rest of the insight too

Reply:
👍